components/package-template/src/etc/clp-schema.template.txt

```diff
@@ -0,0 +1,62 @@
+// Delimiters
+delimiters: \t\r\n!"#$%&'\(\)\*,:;<>?@\[\]\^_`\{\|\}~
+
+// Timestamps (using the `timestamp` keyword)
+// E.g. 2015-01-31T15:50:45.392
+timestamp:\d{4}\-\d{2}\-\d{2}T\d{2}:\d{2}:\d{2}.\d{3}
+// E.g. 2015-01-31T15:50:45,392
+timestamp:\d{4}\-\d{2}\-\d{2}T\d{2}:\d{2}:\d{2},\d{3}
+// E.g. [2015-01-31T15:50:45
+timestamp:\[\d{4}\-\d{2}\-\d{2}T\d{2}:\d{2}:\d{2}
+// E.g. [20170106-16:56:41]
+timestamp:\[\d{4}\d{2}\d{2}\-\d{2}:\d{2}:\d{2}\]
+// E.g. 2015-01-31 15:50:45,392
+// E.g. INFO [main] 2015-01-31 15:50:45,085
```

Comment:
Sorry, I just realized some of these comments will be confusing to users since the current schema support doesn't handle any tokens before the timestamp. Can we delete this line?